iotic-spacectl/iotic-spacectl-intro/step3.md

```diff
@@ -22,27 +22,24 @@ To model the feed you first run:
 
 `./iotic-spacectl modelfeed`{{execute}}
 
-Then simply answer the questions to create some metadata about our feed:
+Then simply answer the questions in the terminal to create some metadata about our feed:
 
-Comments: "Number of occupants in our house"
-Labels: "occupancy"
-
-StoreLastValue: "true" - this allows people looking at our twin to automatically get the last value of our occupancy feed,
+* `Comments`: "Number of occupants in our house"
+* `Labels`: "occupancy"
+* `StoreLastValue`: "true" - this allows people looking at our twin to automatically get the last value of our occupancy feed,
 even if they weren't listening to the feed when we last published to it. The last value of the feed is stored.
+* `Tags`: "occupancy"
+* `Values`:
+* `Label`: "occupancy"
+* `Unit`: "https://data.iotics.com/people"
```

Comment:
This url is just one I made up. We should probably point to a valid unit that exists? But I'm not sure where to find one